src/NLog/Targets/MailTarget.cs

```diff
@@ -280,8 +280,7 @@ public Layout Body
         /// Gets or sets the port number that SMTP Server is listening on.
         /// </summary>
         /// <docgen category='SMTP Options' order='15' />
-        [DefaultValue(25)]
```

Comment:
Isn't the defaultValue attribute working now?

Reply:
I was just assuming it didn't work. Will try and use it anyway.

Reply:
Unit tests seems happy, so have added them back. Lets see what the build says. Recall there are some assembly-wide tests that are only executed on the build-server.